Heart/Season/Infrastructure/Models/Season.php

```diff
@@ -17,30 +17,30 @@ class Season extends Model
 
     protected $table = 'seasons';
 
-   protected $fillable = [
-       'id',
-       'name',
-       'description',
-       'messages_count',
-       'participants_count',
-       'meeting_count',
-       'badges_count',
-       'started_at',
-       'ended_at',
-   ];
-
-   public function badges(): HasMany
-   {
-       return $this->hasMany(Badge::class);
-   }
-
-   public function meetings(): HasMany
-   {
-       return $this->hasMany(Meeting::class);
-   }
-
-    protected static function newFactory(): SeasonFactory
+    protected $fillable = [
+        'id',
+        'name',
+        'description',
+        'messages_count',
+        'participants_count',
+        'meeting_count',
+        'badges_count',
+        'started_at',
+        'ended_at',
+    ];
+
+    public function badges(): HasMany
     {
-        return SeasonFactory::new();
+        return $this->hasMany(Badge::class);
     }
+
+    public function meetings(): HasMany
+    {
+        return $this->hasMany(Meeting::class);
+    }
+
+     protected static function newFactory(): SeasonFactory
+     {
+         return SeasonFactory::new();
+     }
 }
```

Comment:
eu tinha feito algo errado nessa model?

Reply:
Não. Essa é uma função nova que eu implementei pra poder testar coisas novas. Se você for ver, nenhuma função que você implementou era de criação e já tinha seasons por padrão nos seeder então não tinha necessidade até então.

Sua contribuição tá dahora primo.